ql/src/java/org/apache/hadoop/hive/ql/QueryStats.java

```diff
@@ -0,0 +1,34 @@
+package org.apache.hadoop.hive.ql;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * QueryStats.
+ *
+ */
+public class QueryStats {
+  public long queryStart;
```

Comment:
make all data fields in this object **PRIVATE** and add accessors.

Reply:
done